src/libtiled/gidmapper.cpp

```diff
@@ -38,6 +38,7 @@ using namespace Tiled;
 const int FlippedHorizontallyFlag   = 0x80000000;
 const int FlippedVerticallyFlag     = 0x40000000;
 const int FlippedAntiDiagonallyFlag = 0x20000000;
+const int FlippedLeftHexAntiDiagonallyFlag = 0x10000000;
```

Comment:
I don't entirely understand the name or behavior of this flag.

`FlippedAntiDiagonally` meant basically swapping the x/y axis, which was equivalent to flipping vertically and then rotating by 90 degrees. In hexagonal mode, this has changed to mean flipping vertically and then rotating by 60 degrees clockwise (which can no longer be described as an anti-diagonal flip).

`FlippedLeftHexAntiDiagonallyFlag` seems to mean to flip vertically and then rotating by 60 degrees counter-clockwise (which I guess is where the `Left` comes from).

What I'm wondering about is since these flags are not exactly diagonal flips, whether it is useful in any way to still implement them like that. Wouldn't it be simpler if we instead had:
```
const int Rotated60Flag = 0x20000000;   // a duplicate of FlippedAntiDiagonallyFlag for readability
const int Rotated120Flag = 0x10000000;
```
This way things should be easier to understand:

* 60: Rotated60
* 120: Rotated120
* 180: FlippedHorizontally | FlippedVertically
* 240: Rotated60 | FlippedHorizontally | FlippedVertically
* 300: Rotated120 | FlippedHorizontally | FlippedVertically

I hope you don't mind to adjust the patch accordingly.